src/UserService.Mappers/RequestsMappers/UserRequestMapper.cs

```diff
@@ -27,7 +28,14 @@ public DbUser Map(UserRequest value)
                 Status = value.Status,
                 AvatarFileId = value.AvatarFileId,
                 IsActive = value.IsActive,
-                IsAdmin = value.IsAdmin
+                IsAdmin = value.IsAdmin,
+                Connections = value.Connection?.Select(x => new DbConnection
+                {
+                    Id =Guid.NewGuid(),
+                    Type = (int)x.Type,
+                   Value = x.Value,
+                   UserId = value.Id.Value
```

Comment:
you need to make the same indents for properties

Reply:
fixed indents